linode/manager
# M3-2465 Update Auth Code

## Description

Make the auth code look like all our other redux code

### Todo
- [x] Make AuthenticationWrapper.test.tsx work
- [x] Make OAuth.test.tsx work
- [ ] Make an API request to invalidate token upon logout (this will be handled in another PR)

### To Test
1. Make sure that local storage and redux state is being set upon logging in
2. Clear local storage and refresh the page - the app should redirect you to login, then re-set local storage and redux state
3. Using a proxy like Charles, test 401 responses - the app should revalidate you and update local storage and redux state
4. Make sure there are no cases where the user can navigate the app without first being authed
